docs/user-guide/installation/index.rst

```diff
@@ -0,0 +1,124 @@
+Installation
+============
+
+Pulp operates with three main components that get installed potentially on different
+machines.
+
+Server
+  This is the main application server that stores data and distributes content.
+
+Agent
+  This component runs on consumers and communicates with the server to provide remote content
+  management.
+
+Client
+  This is a command line component that comes as two pieces: admin-client,
```

Comment:
If you read the next line, it says "which manages the server". I didn't write this text, it's just moved from the original single file that this all came from.